cypress/e2e/funeral-directors/list-management.feature

```diff
@@ -4,7 +4,7 @@ Feature: List management actions
 		Given I am logged in as a "Administrator"
 		And A "funeralDirectors" list exists for Eurasia
 		And there are these list items
-			| contactName | organisationName       | emailAddress               | status    | isPublished |  service         |
+			| contactName | organisationName       | emailAddress               | status    | isPublished | service          |
```

Comment:
can the smoke tests include the isAnnualReview field as well so the OUT_FOR_ANNUAL_REVIEW scenario can be tested?

Reply:
I thought it made sense to do this in the `cypress/e2e/funeral-directors/list-management.feature` file instead of the funeral-directors specific one. But happy to put the changes in there as well 🤷🏿

Reply:
cypress/e2e/management-list-item-actions.feature

OUT_FOR_ANNUAL_REVIEW won't be a real status. it will be `OUT_WITH_PROVIDER` + `isAnnualReview`. 

you can duplicate `Julia law` and add an additional column in the table `isAnnualReview` true

Reply:
oops, you've done it in an update, ignore